src/components/apps/launch/AppInfo.js

```diff
@@ -97,6 +108,28 @@ const UnavailableMsg = ({ app, hasDeprecatedParams, baseId }) => {
                 }}
             />
         );
+    } else if (computeLimitExceeded) {
+        console.log("formatting the message");
```

Comment:
This can probably go away before merge, I guess

Reply:
Ah, I missed that one. Thanks!